ci-operator/step-registry/cucushift/installer/rehearse/azure/mag/ipi/fullyprivate/provision/cucushift-installer-rehearse-azure-mag-ipi-fullyprivate-provision-chain.yaml

```diff
@@ -0,0 +1,26 @@
+chain:
+  as: cucushift-installer-rehearse-azure-mag-ipi-fullyprivate-provision
+  steps:
+  - ref: azure-provision-resourcegroup
+  - ref: azure-provision-vnet
+  - ref: azure-provision-nat
+  - chain: azure-provision-bastionhost
+  - ref: proxy-config-generate
+  - chain: ipi-conf-azuremag
+  - ref: ipi-conf-azure-provisionednetwork
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-install
```

Comment:
Now we need to enable sourcatalog in all workflow, so can you add `- ref: enable-qe-catalogsource` here.